resources/oracle_install.rb

```diff
@@ -27,7 +27,7 @@
 property :checksum, String, regex: /^[0-9a-f]{32}$|^[a-zA-Z0-9]{40,64}$/
 property :md5, String, regex: /^[0-9a-f]{32}$|^[a-zA-Z0-9]{40,64}$/
 property :app_home, String
-property :app_home_mode, Integer, default: 0755
+property :app_home_mode, Integer, default: 0o755
```

Comment:
is this a typo?

Reply:
no this is an octal value, this was updated automatically by running rubocop automatically, I can revert if we want.

Reply:
We should stick with what cookstyle sets here.

Reply:
per https://github.com/chef/cookstyle/blob/ff76052f3e8f6f7a378f8d7e0fbcdcd0cb99ad26/config/upstream.yml#L3640-L3648 (thx @teknofire) either is acceptable.